src/shared/externalophandler.cpp

```diff
@@ -60,30 +54,13 @@ bool ExternalOpHandler::request(Op op) {
     }
   }
 
-  switch (op) {
-    case OpAbout:
-      vpn->requestAbout();
-      break;
-    case OpActivate:
-      TaskScheduler::deleteTasks();
-      TaskScheduler::scheduleTask(
-          new TaskControllerAction(TaskControllerAction::eActivate));
-      break;
-    case OpDeactivate:
-      TaskScheduler::deleteTasks();
-      TaskScheduler::scheduleTask(
-          new TaskControllerAction(TaskControllerAction::eDeactivate));
-      break;
-    case OpQuit:
-      vpn->controller()->quit();
-      break;
-
-    // List of no-op requests:
-    case OpNotificationClicked:
-      [[fallthrough]];
-    case OpCloseEvent:
-      break;
-  }
+  Q_ASSERT(m_ops.contains(op));
```

Comment:
When would `m_ops` not contain `op`? Should we actually handle that?

Reply:
Sure, but Q_ASSERT only crashes in debug mode. So I production this will won't crash or will crash weirdly, no?